src/fetchCalls.js

```diff
@@ -1,22 +1,53 @@
 const fetchCalls = {
-  getUsers = () => {
-    return fetch('url')
+  getUsers() {
+    // return fetch('http://localhost:3001/api/v1/users')
+    return fetch('https://sniff-api.herokuapp.com/api/v1/users')
       .then(response => {
-        this.checkResponse()
+        this.checkResponse(response)
         return response.json()
       })
-      .then(data => data.users)
+      // .then(data => data.users)
       .catch(error => {
         console.error(error)
         return error.message
       })
   },
 
-  // modifyAppointments = () => {}
+  getSingleUser(userId) {
+    // return fetch(`http://localhost:3001/api/v1/users/${userId}`)
+    return fetch(`https://sniff-api.herokuapp.com/api/v1/users/${userId}`)
+      .then(response => {
+        this.checkResponse(response)
+        return response.json()
+      })
+  },
+
+  getDetailedAppointments(userId) {
+    // return fetch(`http://localhost:3001/api/v1/playdates${userId}`)
+    return fetch(`https://sniff-api.herokuapp.com/api/v1/playdates${userId}`)
+      .then(response => {
+        this.checkResponse(response)
+        return response.json()
+      })
+  },
 
-  // ? getAppointments ?
+  postAppointment(newPlaydates) {
+    // return fetch('http://localhost:3001/api/v1/playdates', {
+    return fetch('https://sniff-api.herokuapp.com/api/v1/playdates', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(newPlaydates)
+    })
+    .then(response => {
+      this.checkResponse(response)
+      console.log(response)
+      return response.json()
+    })
+  },
 
-  checkResponse = () => {
+  checkResponse(response) {
     if (!response.ok) {
       if (response.status === 422 || response.status === 403) {
```

Comment:
Just curious, do you have this logic built into your backend API to return these status codes given certain situations? Let us know if you need help around this idea.